app/models/spotlight/resource.rb

```diff
@@ -30,10 +29,23 @@ def self.class_for_resource r
     end
 
     def to_solr
-      {
-        spotlight_resource_id_ssim: "#{(type.tableize if type) || self.class.to_s.tableize }:#{id}",
-        spotlight_resource_url_ssim: url
-      }
+      exhibit.solr_data.merge({
+        :"#{Spotlight::Engine.config.solr_fields.prefix}spotlight_resource_id#{Spotlight::Engine.config.solr_fields.string_suffix}" => "#{(type.tableize if type) || self.class.to_s.tableize }:#{id}",
```

Comment:
I realized we were hard-coding `_ssim` here, so I've fixed that. I also updated `spotlight_resource_url_ssm` to be `spotlight_resource_url_ssim` to avoid introducing another field type. I don't think that's a problem, right?